rules.md

```diff
@@ -76,17 +54,16 @@ To ensure that the Incubator's processes and information remain fully open-sourc
 5. All people's work and rewards are public and verifiable
 6. All rules governing the Incubator and decisions on how to maintain those rules are public
 
-The only exception would be user credentials, i.e. their login passwords (which won't be needed when the Incubator App moves onto Dash Platform)
-
 ## 1.5 Users
 
-There are several levels to classification of Incubator App users as follows:
+There are several types of Incubator users, as follows:
 
-- Users - anyone who accesses the Incubator App or its underlying data
-- Members - any user who has signed up to create an account on the App
+- Users - anyone who accesses the Incubator or its underlying data
+- Members - any user who has signed up to the Incubator
 - [Contributors](#3-contributing) - Members who create output for the Incubator
-- [Admins](#4-admins) - Members with rights to create and manage Bounties, and earn commission from Task rewards
-- [Proposal Owner (PO)](#53-proposal-owner) - an Admin who is designated to raise Proposals to the Dash network on behalf of the Incubator 
+- [Admins](#4-admins) - Members with rights to create and manage Bounties
+- [Strategists](#46-strategists) - Admins who manage and report on Budgets
+- [Lead Strategist](#47-lead-strategist) - The strategist who raises Proposals to the Dash network on behalf of the Incubator 
```

Comment:
Is there another document that could be linked here or more info on the responsibility of the Lead Strategist role in relation to Strategists? A more defined explanation of how disagreements or similar issues are handled?

Reply:
There's Section 4.7 (linked to), and section 4.5 about governance, but did you mean something more?

I'd like the rules to be as lean as possible, but happy to add any important details that are needed (want to suggest something specific).

Reply:
That should be sufficient, then (don't know how to resolve this another way)